bazel_tools/grpc-haskell-core-upgrade.patch

```diff
@@ -0,0 +1,70 @@
+diff --git a/cbits/grpc_haskell.c b/cbits/grpc_haskell.c
+index ebbcbb1..236c73e 100644
+--- a/cbits/grpc_haskell.c
++++ b/cbits/grpc_haskell.c
+@@ -504,8 +504,7 @@ grpc_auth_metadata_processor* mk_auth_metadata_processor(
+ 
+ grpc_call_credentials* grpc_metadata_credentials_create_from_plugin_(
+   grpc_metadata_credentials_plugin* plugin){
+-
+-  return grpc_metadata_credentials_create_from_plugin(*plugin, NULL);
++  return grpc_metadata_credentials_create_from_plugin(*plugin, GRPC_PRIVACY_AND_INTEGRITY, NULL);
```

Comment:
this technically could be make configurable but the python bindings also don’t make it configurable atm. This is the highest security level which afaict was the default before as well.